app/helpers/cable_ready/view_helper.rb

```diff
@@ -27,6 +26,10 @@ def updates_for_if(condition, *keys, **options, &block)
       end
     end
 
+    def cable_car
+      CableReady::CableCar.instance
+    end
+
```

Comment:
Does it even make sense that `cable_car` is available in the View scope? What would you use this for?

Reply:
No clue but I don’t wanna break stuff

Reply:
Probably some folks use it in controllers etc?

Reply:
Well, I don’t know 

Anyway I don’t wanna remove stuff without understanding the ramifications